packages/preview-middleware-client/src/adp/quick-actions/fe-v2/lr-enable-semantic-date-range-filter-bar.ts

```diff
@@ -25,7 +26,14 @@ export class ToggleSemanticDateRangeFilterBar
     readonly forceRefreshAfterExecution = true;
     private isUseDateRangeTypeEnabled = false;
 
-    initialize(): void {
+    async initialize(): Promise<void> {
+        const version = await getUi5Version();
+        const isUI5VersionNotSupported =
+            isLowerThanMinimalUi5Version(version, { major: 1, minor: 128 }) ||
+            (version.major === 1 && (version.minor === 96 || version.minor === 108 || version.minor === 120));
```

Comment:
Shouldn't `1.96`, `1.108` and `1.120` should be also supported? Currently it looks like all the versions below `1.128` are not supported.

Reply:
adapted the logic with test (with exact version 1.96). the condition was not handling it correctly.